CMakeLists.txt

```diff
@@ -23,6 +23,10 @@ if(IMPORT_DEPTH_CAM_FW AND NOT INTERNET_CONNECTION)
     set(IMPORT_DEPTH_CAM_FW OFF)
 endif()
 
+set(BUILD_IPDEV           ON CACHE BOOL "Enable IP device support")
+set(BUILD_IPDEV_UNITTESTS ON CACHE BOOL "Enable unittests for IP device")
+set(BUILD_IPDEV_STATS     ON CACHE BOOL "Enable statistics for IP device")
```

Comment:
What does it do? Why would the user enable it?

Reply:
BUILD_IPDEV_UNITTESTS will be removed. BUILD_IPDEV_STATS should turn on performance counters to allow delay and latency estimation.

Reply:
Done